server/routes/termdb.termsbyids.ts

```diff
@@ -0,0 +1,57 @@
+// import { gettermbyidRequest, gettermbyidResponse } from '#shared/types/routes/termdb.termbyid'
```

Comment:
please define the request and response types at `server/shared/types/routes/` it's easy, refer to existing examples